### PR TITLE
Dịch "dropout_vn" - Phần 5

### LINKED ISSUE #947: Dịch "dropout_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/dropout_vn.md#L312-L407

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by nguyenduydu**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #947

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 1]
```diff
@@ -324,7 +324,10 @@ Below we set it to 0.2 and 0.5 for the first and second hidden layer respectivel
 By using the `is_training` function described in :numref:`sec_autograd`, we can ensure that dropout is only active during training.
 -->
 
-*dịch đoạn phía trên*
+Mô hình bên dưới áp dụng dropout lên đầu ra của mỗi tầng ẩn (theo sau hàm kích hoạt).
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Mô hình bên dưới áp dụng dropout cho đầu ra của mỗi tầng ẩn (theo sau hàm kích hoạt).
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 2]
```diff
@@ -324,7 +324,10 @@ Below we set it to 0.2 and 0.5 for the first and second hidden layer respectivel
...
+Ta có thể đặt các giá trị xác suất dropout cho mỗi tầng một cách riêng biệt. Một xu hướng chung là đặt một xác suất dropout thấp hơn cho tầng ở gần với tầng đầu vào hơn.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Ta có thể đặt các giá trị xác suất dropout riêng biệt cho mỗi tầng. Một xu hướng chung là đặt xác suất dropout thấp hơn cho tầng ở gần với tầng đầu vào hơn.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 3]
```diff
@@ -371,15 +374,16 @@ d2l.train_ch3(net, train_iter, test_iter, loss, num_epochs,
...
+## Triển khai súc tích
 
 <!--
 Using Gluon, all we need to do is add a `Dropout` layer (also in the `nn` package) after each fully-connected layer, passing in the dropout probability as the only argument to its constructor.
 During training, the `Dropout` layer will randomly drop out outputs of the previous layer (or equivalently, the inputs to the subsequent layer) according to the specified dropout probability.
 When MXNet is not in training mode, the `Dropout` layer simply passes the data through during testing.
 -->
 
-*dịch đoạn phía trên*
+Bằng việc sử dụng Gluon, tất cả những gì ta cần là thêm một tầng `Dropout` (cũng nằm trong gói `nn`) vào sau mỗi tầng kết nối đầy đủ và truyền vào xác suất dropout như là đối số duy nhất của hàm khởi tạo.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Bằng việc sử dụng Gluon, tất cả những gì ta cần làm là thêm một tầng `Dropout` (cũng nằm trong gói `nn`) vào sau mỗi tầng kết nối đầy đủ và truyền vào xác suất dropout, đối số duy nhất của hàm khởi tạo.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 4]
```diff
@@ -371,15 +374,16 @@ d2l.train_ch3(net, train_iter, test_iter, loss, num_epochs,
...
+Trong quá trình huấn luyện, hàm `Dropout` sẽ bỏ đi một cách ngẫu nhiên một số đầu ra của tầng trước (hay tương đương với đầu vào của tầng tiếp theo) dựa trên xác suất dropout được định nghĩa trước đó.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Trong quá trình huấn luyện, hàm `Dropout` sẽ bỏ ngẫu nhiên một số đầu ra của tầng trước (hay tương đương với đầu vào của tầng tiếp theo) dựa trên xác suất dropout được định nghĩa trước đó.
```

**Comment by nguyenduydu** [comment]
@minhduc0711  Cám ơn bạn đã góp ý.

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 5]
```diff
@@ -315,7 +315,7 @@ for param in params:
 ### Defining the Model
 -->
 
-### *dịch tiêu đề phía trên*
+### Định nghĩa mô hình
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
### Định nghĩa Mô hình
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 6]
```diff
@@ -347,13 +350,13 @@ def net(X):
 ### Training and Testing
 -->
 
-### *dịch tiêu đề phía trên*
+### Huấn luyện và kiểm tra
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
### Huấn luyện và Kiểm tra
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 7]
```diff
@@ -483,7 +487,7 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 5 -->
-*
+* Nguyễn Duy Du
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
* Nguyễn Duy Du
* Phạm Minh Đức
```
